bundle/compliance/policy/aws_s3/ensure_block_public_access.rego

```diff
@@ -7,17 +7,38 @@ import future.keywords.in
 
 default rule_evaluation = false
 
+# If we got public access block config for both account and bucket
 rule_evaluation {
+	not data_adapter.public_access_block_configuration == null
+	not data_adapter.account_public_access_block_configuration == null
 	assert.some_true([data_adapter.public_access_block_configuration.BlockPublicAcls, data_adapter.account_public_access_block_configuration.BlockPublicAcls])
 	assert.some_true([data_adapter.public_access_block_configuration.BlockPublicPolicy, data_adapter.account_public_access_block_configuration.BlockPublicPolicy])
 	assert.some_true([data_adapter.public_access_block_configuration.IgnorePublicAcls, data_adapter.account_public_access_block_configuration.IgnorePublicAcls])
 	assert.some_true([data_adapter.public_access_block_configuration.RestrictPublicBuckets, data_adapter.account_public_access_block_configuration.RestrictPublicBuckets])
 }
 
+# If we got only account-level public access block config
+rule_evaluation {
+	not data_adapter.account_public_access_block_configuration == null
+	data_adapter.public_access_block_configuration == null
+	data_adapter.account_public_access_block_configuration.BlockPublicAcls == true
+	data_adapter.account_public_access_block_configuration.BlockPublicPolicy == true
+	data_adapter.account_public_access_block_configuration.IgnorePublicAcls == true
+	data_adapter.account_public_access_block_configuration.RestrictPublicBuckets == true
```

Comment:
It kind of repeats itself, can we use a util function to make the rule a bit more concise?
As we want to push it today, we can decide to change it in the future if it takes too much time.